### PR TITLE
New functionality for DSC Extension - Publish-AzureVMDscConfiguration cmdlet

### PR DESCRIPTION
**Pull request by eshaparmar**
This is in continuation to the PR #673 New functionality for DSC Extension - Publish-AzureVMDscConfiguration cmdlet. 
Note that this branch already has got latest from azure branch - release-0.9.6

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@eshaparmar__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (esparmar). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by yugangw-msft** [comment]
http://azuresdkci.cloudapp.net/view/1-AzurePowerShell/job/powershell-on-demand/65/
http://azuresdkci.cloudapp.net/view/1-AzurePowerShell/job/powershell-msi/398/

File: AzurePowershell.Test.targets [hunk 1]
```diff
@@ -26,7 +26,6 @@
     <WebsitesTestDebug>.\src\ResourceManager\Websites\Commands.Websites.Test\bin\Debug\Microsoft.Azure.Commands.Websites.Test.dll</WebsitesTestDebug>
     <RemoteAppTestDebug>.\src\ServiceManagement\RemoteApp\Commands.RemoteApp.Test\bin\Debug\Microsoft.Azure.Commands.RemoteApp.Tests.dll</RemoteAppTestDebug>
     <ServiceManagementTestDebug>.\src\ServiceManagement\Compute\Commands.ServiceManagement.Test\bin\Debug\Microsoft.WindowsAzure.Commands.ServiceManagement.Test.dll</ServiceManagementTestDebug>
-    <ServiceManagementExtensionsTestDebug>.\src\ServiceManagement\Compute\Commands.ServiceManagement.Extensions.Test\bin\Debug\Microsoft.WindowsAzure.Commands.ServiceManagement.Extensions.Test.dll</ServiceManagementExtensionsTestDebug>
```
Comment (yugangw-msft) [inline AzurePowershell.Test.targets]: i assume removing this is expected, right

**Comment by yugangw-msft** [comment]
did you copy the source code from the release branch? rather just branch from the dev branch? If yes, please do not do that, as this will conflict with our integration from master back the dev branch

File: AzurePowershell.Test.targets (hunk 1 above)
Comment (eshaparmar) [inline AzurePowershell.Test.targets]: Yes.

**Comment by eshaparmar** [comment]
Yes, I had to merge the code from release-0.9.6 branch as Dev branch was broken.

**Comment by eshaparmar** [comment]
Yes fake one

**Pull request merged**